### PR TITLE
Remove nth node from the end

### PR DESCRIPTION
**Pull request by grassisgreenerr**
# Remove nth Node from the end  of the linked list Code

### CODE REVIEW

**Review by Rickety-Code — COMMENTED** [review]
Kindly change the file name to match the PascalCase scheme.

File: DP/Trapping Rain Water/trapping_rain_water.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    cin >> n;
+
+    int arr[n];
+    for (int i = 0; i < n; i++)
+        cin >> arr[i];
+
+    cout << maxWater(arr, n);
+
+    return 0;
+}
```
Comment (Rickety-Code) [inline DP/Trapping Rain Water/trapping_rain_water.cpp:47]: Can be removed. In functional programming, it's okay to omit the main method where the flow starts from.

**Comment by Rickety-Code** [comment]
Please have the issues assigned to you before you begin working on them.
-Kindly take reference from issue which are already assigned to other contributors.

File: DP/Trapping Rain Water/trapping_rain_water.cpp (hunk 1 above)
Comment (grassisgreenerr) [reply DP/Trapping Rain Water/trapping_rain_water.cpp]: okay. got it.  i am doing it right now.

**Comment by grassisgreenerr** [comment]
> Please have the issues assigned to you before you begin working on them. -Kindly take reference from issue which are already assigned to other contributors.

okay.

**Pull request closed without merging**